prime-angular-frontend/src/app/modules/enrolment/shared/components/obo-site-form/obo-site-form.component.html

```diff
@@ -24,14 +24,33 @@
 
           <mat-form-field class="w-100">
             <input matInput
-                   placeholder="Health Authority Facility Name"
+                   placeholder="'{{healthAuthorityName}}' Health Authority Facility Name"
```

Comment:
Is there a reason why we're surrounding this with single quotes?  If it is to emphasize the health authority name it doesn't do that visually.

![image](https://user-images.githubusercontent.com/599498/116457480-4f6a9a80-a818-11eb-99d3-14374bf15f7f.png)

If you want to emphasis the health authority name you'll want to add a `mat-label`:

```
<mat-label> 
  <strong>{{ healthAuthorityName }}</strong> Health Authority Facility Name
</mat-label>
```

Reply:
Updated